docs/docs/policies.mdx

```diff
@@ -85,25 +85,35 @@ The Transformer Embedding Dialogue (TED) Policy is described in
 This policy has a pre-defined architecture, which comprises the
 following steps:
 
-* concatenate user input (user intent and entities), previous system actions, slots and active forms for each time
+1. concatenate user input (user intent and entities), previous system actions, slots and active forms for each time
   step into an input vector to pre-transformer embedding layer;
 
-* feed it to transformer;
+2. feed it to transformer;
 
-* apply a dense layer to the output of the transformer to get embeddings of a dialogue for each time step;
+3. apply a dense layer to the output of the transformer to get embeddings of a dialogue for each time step;
 
-* apply a dense layer to create embeddings for system actions for each time step;
+4. apply a dense layer to create embeddings for system actions for each time step;
 
-* calculate the similarity between the dialogue embedding and embedded system actions.
-  This step is based on the [StarSpace](https://arxiv.org/abs/1709.03856) idea.
+5. calculate the similarity between the dialogue embedding and embedded system actions.
+   This step is based on the [StarSpace](https://arxiv.org/abs/1709.03856) idea.
 
-It is recommended to use `state_featurizer=LabelTokenizerSingleStateFeaturizer(...)`
-(see [Featurization of Conversations](./policies.mdx#featurization-conversations) for details).
+It is recommended to use `LabelTokenizerSingleStateFeaturizer`
+(see [Featurization of Conversations](./policies.mdx#featurizers) for details).
+
+```yaml-rasa title="config.yml"
+policies:
+  - name: "TEDPolicy"
+    featurizer:
+    - name: MaxHistoryTrackerFeaturizer
+      max_history: 5
+      state_featurizer:
+        - name: LabelTokenizerSingleStateFeaturizer
+```
```

Comment:
this is a placeholder: this section didn't contain codeblocks and we thought it'd be a great idea to show the user how to configure the policy. But we didn't know what the content should be, please advise!

Reply:
showing how to pass the featurizer is great 👍 

on content, @Ghostvv  does it look good?

Reply:
`LabelTokenizerSingleStateFeaturizer` is deprecated, we have only `SingleStateFeaturizer` now